EpsilonDriverControls/Src/DriverControls.c

```diff
@@ -206,11 +207,7 @@ void sendDriverTask(void const* arg)
 void sendDriveCommandsTask(void const* arg)
 {
     uint32_t prevWakeTime = osKernelSysTick();
-    float busCurrentOut = 1.0f; // Percentage 0 -1 always 100%
-    float motorVelocityOut = 0; // RPM
-    float motorCurrentOut = 0.0f; // Percentage 0 - 1
```

Comment:
`motorCurrentOut` needs to stay outside, because the lines:
`motorCurrentOut = calculateRegenMotorCurrent(regenPercentage, motorCurrentOut);`
Require a saved value of motorCurrentOut to do their calculations.